pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -987,7 +1028,7 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 		err := controllerutil.SetControllerReference(instance, serviceMonitor, r.scheme)
 		return err
 	})
-	if err != nil {
+	if _, isNoKindMatchError := err.(*meta.NoKindMatchError); err != nil && !isNoKindMatchError {
 		return reconcile.Result{}, err
```

Comment:
Is there a reason this and the ones below are not using `r. manageError`?

Reply:
Good spot, thanks -- this is new code that showed up after a rebase & I didn't update it (apart from the fix to ignore if it's a NoKindMatchError). I'll update it :+1: